MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ValidationErrorTemplate.xaml

```diff
@@ -29,6 +30,7 @@
                            Margin="0 2"
                            TextWrapping="Wrap"
                            Text="{Binding CurrentItem.ErrorContent}"
+                           Opacity="{Binding Path=(wpf:ValidationAssist.ValidateeOpacity), ElementName=Placeholder}"
```

Comment:
Would this make more sense to just directly bind to the element rather than introduce an additional attached property?
```suggestion
                           Opacity="{Binding ElementName=Placeholder, Path=AdornedElement.Opacity}"
```

Reply:
Since the `Transitioner` hides the slide by setting `TransitionerSlide.Opacity` to zero, `Opacity` of the elements on that slide does not change. `AdornedElement.Opacity` is also non-zero, so binding it does not hide the validation error message.
Therefore, `ValidationAssist.ValidateOpacity` that has the value of `TransitionerSlide.Opacity` should be used instead of `AdornedElement.Opacity`.